src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -267,6 +267,8 @@ OPENVINO_SUPPRESS_DEPRECATED_START
 OPENVINO_SUPPRESS_DEPRECATED_END
         } else if (key == CONFIG_KEY(LOG_LEVEL) || key == ov::log::level) {
             gnaFlags.log_level = ov::util::from_string(value, ov::log::level);
+        } else if (key == CONFIG_KEY(CACHE_DIR) || key == ov::cache_dir) {
+            cacheDir = value;
```

Comment:
Do we need any validation of the value here?

Reply:
I do not find any validation for CACHE_DIR option in CPU plugins, which is reference implementation. For FIRMWARE_MODEL_IMAGE we also do not perform validation. If empty string is passed as cache directory then this option is threated as disabled.